src/build.d

```diff
@@ -34,15 +34,18 @@ immutable rootDeps = [
     &dmdDefault,
     &runDmdUnittest,
     &clean,
+    &examples,
 ];
 
 void main(string[] args)
 {
     int jobs = totalCPUs;
+    bool calledFromMake = false;
     auto res = getopt(args,
         "j|jobs", "Specifies the number of jobs (commands) to run simultaneously (default: %d)".format(totalCPUs), &jobs,
         "v", "Verbose command output", (cast(bool*) &verbose),
         "f", "Force run (ignore timestamps and always run all tests)", (cast(bool*) &force),
+        "called-from-make", "Calling the build script from the Makefile", &calledFromMake
```

Comment:
Can you utilize the [`.NOTPARALLEL` pseudo-target](https://www.gnu.org/software/make/manual/html_node/Parallel.html) instead?

Reply:
That only applies to the posix.mak file, and I don't think people would appreciate removing the parallelism of the Makefile during this transition. This way both build systems can still be parallel and it works the same for windows and posix.